pkg/cmd/cli/uninstall/uninstall.go

```diff
@@ -104,50 +103,58 @@ func Run(ctx context.Context, client *kubernetes.Clientset, extensionsClient *ap
 		if ns.Status.Phase == corev1.NamespaceTerminating {
 			fmt.Printf("Velero installation namespace %q is terminating.\n", namespace)
 		} else {
-			err = client.CoreV1().Namespaces().Delete(ctx, ns.Name, metav1.DeleteOptions{})
-			if err != nil {
+			if err := kbClient.Delete(context.Background(), ns); err != nil {
 				errs = append(errs, errors.WithStack(err))
 			}
 		}
 	}
 
 	// rolebinding
 	crb := install.ClusterRoleBinding(namespace)
-	if err := client.RbacV1().ClusterRoleBindings().Delete(ctx, crb.Name, metav1.DeleteOptions{}); err != nil {
+	key = kbclient.ObjectKey{Name: crb.Name, Namespace: namespace}
+	if err := kbClient.Get(context.Background(), key, crb); err != nil {
 		if apierrors.IsNotFound(err) {
-			fmt.Printf("Velero installation clusterrolebinding %q does not exist, skipping.\n", crb.Name)
+			fmt.Printf("Velero installation rolebinding %q does not exist, skipping.\n", crb.Name)
 		} else {
 			errs = append(errs, errors.WithStack(err))
 		}
+	} else {
+		if err := kbClient.Delete(context.Background(), crb); err != nil {
+			errs = append(errs, errors.WithStack(err))
+		}
 	}
 
 	// CRDs
 	veleroLabels := labels.FormatLabels(install.Labels())
-	crds, err := extensionsClient.ApiextensionsV1().CustomResourceDefinitions().List(ctx, metav1.ListOptions{
-		LabelSelector: veleroLabels,
-	})
-	if err != nil {
-		errs = append(errs, errors.WithStack(err))
+	crdList := apiextv1beta1.CustomResourceDefinitionList{}
+	opts := kbclient.ListOptions{
+		Namespace: namespace,
+		Raw: &metav1.ListOptions{
+			LabelSelector: veleroLabels,
+		},
 	}
-	if len(crds.Items) == 0 {
-		fmt.Print("Velero CRDs do not exist, skipping.\n")
+	if err := kbClient.List(context.Background(), &crdList, &opts); err != nil {
+		errs = append(errs, errors.WithStack(err))
 	} else {
-		for _, removeCRD := range crds.Items {
-			if err = extensionsClient.ApiextensionsV1().CustomResourceDefinitions().Delete(ctx, removeCRD.ObjectMeta.Name, metav1.DeleteOptions{}); err != nil {
-				err2 := errors.WithMessagef(err, "Uninstall failed removing CRD %s", removeCRD.ObjectMeta.Name)
-				errs = append(errs, errors.WithStack(err2))
+		if len(crdList.Items) == 0 {
+			fmt.Print("Velero CRDs do not exist, skipping.\n")
+		} else {
+			for _, crd := range crdList.Items {
```

Comment:
Rather than using this loop, I think we could simply using the same `LabelSelector` inside of a [`DeleteAllOfOptions`](https://pkg.go.dev/sigs.k8s.io/controller-runtime/pkg/client#DeleteAllOfOptions) object passed into the [`kbclient.DeleteAllOf`](https://pkg.go.dev/sigs.k8s.io/controller-runtime/pkg/client#Writer) function.